modules/suggestions/reactions.ts

```diff
@@ -36,11 +37,42 @@ export async function addToDatabase(thread: AnyThreadChannel): Promise<void> {
 	const defaultEmoji = config.channels.suggestions?.defaultReactionEmoji;
 	const message = await thread.fetchStarterMessage().catch(() => void 0);
 	const count = (defaultEmoji?.id && message?.reactions.resolve(defaultEmoji.id)?.count) || 0;
+	const suggestionData = getSuggestionData(thread);
+	const data = suggestionsDatabase.data as {
+		answer: typeof suggestionAnswers[number];
+		author: Snowflake;
+		count: number;
+		id: Snowflake;
+		title: number | string;
+	}[];
+	const dupes = await findDuplicates(suggestionData, data);
+	if (dupes.length == 0) return;
+	const links = dupes.map((dupe) => {
+		return `<#${dupe.id}> ${dupe.answer}`;
+	});
+	await thread.send(`
+	## Possible dupes found:\n${links.join("\n")}
+	`);
+	suggestionsDatabase.data = [...suggestionsDatabase.data, { ...suggestionData, count }];
+}
 
-	suggestionsDatabase.data = [
-		...suggestionsDatabase.data,
-		{ ...getSuggestionData(thread), count },
-	];
+export async function findDuplicates(
```

Comment:
This function is only used once, and only has one line. Just inline it. That'll also fix the readonly error